pages/common/nginx.md

```diff
@@ -8,4 +8,4 @@
 
 - Start server with custom config and custom relative path base directory
 
-`nginx -c {{path to config}} -p {{relative path base dir}}`
+`nginx -c {{config_file}} -p {{relative/path/to/base/dir}}`
```

Comment:
Sure: `nginx -c /opt/app/nginx.conf -p /opt/app`. This will start an nginx server based on the config `/opt/app/nginx.conf`. All relative paths within that config file will be resolved against `/opt/app`.